### ISSUE TITLE
Stats globales : Ajout du nombre d'organismes ?

### ISSUE THREAD

**Issue by DonovanMaillard** [PnX-SI/GeoNature#1207]
Bonjour, 

Je trouverais intéressant que les stats globales incluent le nombre d'organismes impliqués dans l'acquisition des données. 

Est-ce que ça vous semble pertinent ?

**Comment by TheoLechemia** [comment]
Oui, plutôt que le nombre d'observateur qui est complètement approximatif

**Comment by DonovanMaillard** [comment]
Ok ! Du coup en comptant le nombre distinct d'acteurs dans la cor_dataset_actor et en indiquant "X acteurs" ? :)

**Comment by TheoLechemia** [comment]
Oui !

**Comment by camillemonchicourt** [comment]
Les acteurs c'est un concept un peu fourre-tout. Ca mélange des personnes et des organismes. Ce sont ni vraiment les observateurs, ni vraiment les organismes.
Donc je prendrai que les organismes.
Et encore à voir si ça veut vraiment dire quelque chose.

**Comment by DonovanMaillard** [comment]
Pour pas mal de JDD notamment au niveau national, les organismes ne sont pas précisés (par exemple pour CardObs je crois). 

Mais à voir en effet... perso je garderais quand meme l'acteur dans son ensemble (comme pour les taxons, qui sont tout aussi fourre tout)

**Comment by DonovanMaillard** [comment]
Intégré partiellement sur mon fork (je ne récupère que les organismes des JDD pour le moment, pas du CA lié). 

En revanche ca m'a permis de remarquer qu'il y a une incohérence entre le nombre de données renvoyé dans les stats globales, et ce que l'utilisateur voit réellement lorsqu'il requête tout dans la syntèse. 

Les stats globales dans mon cas indiquent un peu moins de données que ce que lasynthèse propose à l'utilisateur. Le filtrage  ou calcul ne semble pas s'appuyer sur le même tri.
